From 616637d4ce49d67bba8478c108b0b02fa2ab7544 Mon Sep 17 00:00:00 2001
Message-Id: <patch-v4-01.34-616637d4ce4-20211117T101025Z-avarab@gmail.com>
In-Reply-To: <cover-v4-00.34-00000000000-20211117T101025Z-avarab@gmail.com>
References: <cover-v3-00.23-00000000000-20211116T114334Z-avarab@gmail.com>
	<cover-v4-00.34-00000000000-20211117T101025Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 26 Mar 2021 18:37:42 +0100
Subject: [PATCH v4 01/34] gc docs: reword and move "CONFIGURATION" section
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Reword the blurb I added in b6a8d09f6d8 (gc docs: include the "gc.*"
section from "config" in "gc", 2019-04-07) to be spell out more
clearly how this inclusion happens. This is in preparation for moving
that text to an included file and using it more widely.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Documentation/git-gc.txt | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/Documentation/git-gc.txt b/Documentation/git-gc.txt
index 853967dea01..afb2fef1d04 100644
--- a/Documentation/git-gc.txt
+++ b/Documentation/git-gc.txt
@@ -103,14 +103,6 @@ more time, and the resulting space/delta optimization may or may not
 be worth it. Not using this at all is the right trade-off for most
 users and their repositories.
 
-CONFIGURATION
--------------
-
-The below documentation is the same as what's found in
-linkgit:git-config[1]:
-
-include::config/gc.txt[]
-
 NOTES
 -----
 
@@ -150,6 +142,14 @@ HOOKS
 The 'git gc --auto' command will run the 'pre-auto-gc' hook.  See
 linkgit:githooks[5] for more information.
 
+CONFIGURATION
+-------------
+
+Everything below this line in this section is selectively included
+from the linkgit:git-config[1] documentation. The content is the same
+as what's found there:
+
+include::config/gc.txt[]
 
 SEE ALSO
 --------
-- 
2.34.0.796.g126c2ced91c

